Homework 9 - Chapter 10 - Nguyen Chinh Quan - 220066

In [16]:
import pandas as pd
import math
import numpy as np

10.14.

a. mean1: population mean rotation speed of Sonicare Elite, mean2: population mean rotation speed of Oral-B

H0: mean1 - mean2 = delta0, Ha: mean1 - mean2 > 0

alpha = 0.05 -> z0.05 = 1.64

In [17]:
n1_14 = 23
n2_14 = 25
sam_mean1_14 = 7992.2
sam_mean2_14 = 7988.2
pop_var1_14 = 1260.25
pop_var2_14 = 1697.44
z_14 = ((sam_mean1_14 - sam_mean2_14) - 0)/math.sqrt(pop_var1_14/n1_14 + pop_var2_14/n2_14)
print(z_14)


0.3611216280775699


0.36 < 1.64, therefore there is no evidence to suggest that the Sonicare Elite has a greater population mean rotation speed than the Oral-B.

b.

Using online calcultor, the p-value is 0.359424.

10.18.

In [18]:
data_18 = pd.read_excel(r"D:\stat\Tex-_Datasets\Chapter 10\ex10-18taxiride.xls")
print(data_18["San Diego (min)"].mean(), data_18["Phoenix (min)"].mean(), len(data_18["San Diego (min)"]), len(data_18["Phoenix (min)"]))

20.322222222222223 14.777777777777779 27 27


alpha = 0.01 -> alpha/2 = 0.005 -> z_0.005 = -2.3263

mean1: mean taxi ride time in San Diego, mean2: mean taxi ride time in Phoenix

H0: mean1 - mean2 = 0, Ha: mean1 - mean2 != 0

In [19]:
sam_std1_18 = 6.2
sam_std2_18 = 4.9
sam_mean1_18 = 20.3
sam_mean2_18 = 14.8
z_18 = ((sam_mean1_18 - sam_mean2_18) - 0) / math.sqrt(sam_std1_18**2/27 + sam_std2_18**2/27)
print(z_18)

3.616415735257241


3.62 > -(-2.3263), therefore there is evidence to suggest that the mean taxi ride time is different in San Diego and Phoenix.

10.45.

a.

mean1: population mean deviations from perfect flatness for Tire Rack, mean2: population mean deviations from perfect flatness for JC Whitney

H0: mean1 - mean2 = 0, Ha: mean1 - mean2 != 0

alpha = 0.05 -> alpha/2 = 0.025, n1+n2-2 = 23 -> t_alpha/2,23 = 2.069

In [20]:
n1_45 = 11
n2_45 = 14
sam_mean1_45 = 26.74
sam_mean2_45 = 29.53
sam_std1_45 = 8.31
sam_std2_45 = 6.85
s_squared_p_45 = ((n1_45 - 1)*sam_std1_45**2+(n2_45 - 1)*sam_std2_45**2)/23 
t_45 = ((sam_mean1_45 - sam_mean2_45) - 0)/math.sqrt(s_squared_p_45 * (1/n1_45 + 1/n2_45))
print(t_45)

-0.9208616064461812


|-0.92| = 0.92 < 2.069, therefore there is no evidence to suggest a difference in population mean deviations from perfect flatness for these two rotor brands 

b. Using t-table for df=23, we can see that the bounds for p-value are 0.2 and 0.4 (0.2 <= p <= 0.4). 

10.49.

a.

mean1: population mean sagittal diameter of women's biceps tendons, mean2: population mean sagittal diameter of men's biceps tendons

H0: mean1 - mean2 = 0, Ha: mean1 - mean2 != 0

alpha = 0.01 -> alpha/2 = 0.005, n1+n2-2 = 100 -> t_0.005,100 = 2.626

In [21]:
n1_49 = 54
n2_49 = 48
sam_mean1_49 = 2.5
sam_mean2_49 = 2.8
sam_std1_49 = 0.49
sam_std2_49 = 0.49
s_squared_p_49 = ((n1_49 - 1)*sam_std1_49**2+(n2_49 - 1)*sam_std2_49**2)/100 
t_49 = ((sam_mean1_49 - sam_mean2_49) - 0)/math.sqrt(s_squared_p_49 * (1/n1_49 + 1/n2_49))
print(t_49)

-3.0863316137760197


|-3.09| = 3.09 > 2.626, therefore there is evidence to suggest that the population mean sagittal diameter of women’s biceps tendons is different
from that of men’s biceps tendons.

b.

alpha = 0.05 -> alpha/2 = 0.025 -> t_0.025,100 = 1.984

In [22]:
lower_49 = (sam_mean1_49 - sam_mean2_49) - 1.984 * math.sqrt(sam_std1_49**2/n1_49 + sam_std2_49**2/n2_49)
upper_49 = (sam_mean1_49 - sam_mean2_49) + 1.984 * math.sqrt(sam_std1_49**2/n1_49 + sam_std2_49**2/n2_49)
print(lower_49, upper_49)

-0.4928503072525614 -0.10714969274743824


95% confidence interval for the difference in population mean sagittal diameters is (-0.493, -0.107).

10.79.

In [23]:
data_79 = pd.read_excel(r"D:\stat\Tex-_Datasets\Chapter 10\ex10-79glock.xls")
data_79

Velocity Before (ft/s)  Velocity After (ft/s)
0                    1505                   1625
1                    1419                   1511
2                    1504                   1459
3                    1494                   1441
4                    1510                   1472
5                    1506                   1521

a. Same handguns are used.

b.

H0: uD = 0, Ha: uD < 0

alpha = 0.01, n=6 -> t_0.01,5 = 3.365

In [24]:
d_79 = (np.array(data_79["Velocity Before (ft/s)"]) - np.array(data_79["Velocity After (ft/s)"])).sum()/6
sd_79 = math.sqrt(((np.array(data_79["Velocity Before (ft/s)"]) - np.array(data_79["Velocity After (ft/s)"]) - d_79)**2).sum()/5)
t_79 = (d_79 - 0) / (sd_79 / math.sqrt(6))
print(t_79)

-0.496562465309264


-0.497 > -3.365, therefore there is no evidence to suggest that the population mean muzzle velocity of a clean gun is greater than the population mean muzzle velocity of a dirty gun.

c. Using the t-table for df = 5, we can see that the bound for p-value are 0.25 and 0.5.

10.82.

In [25]:
d_82 = 1.4923
sd_82 = 2.8097
n_82 = 26

a. alpha = 0.05 -> t_0.05,25 = 1.708

H0: uD = 0, Ha: uD > 0

In [26]:
t_82 = (d_82 - 0) / (sd_82 / math.sqrt(26))
print(t_82)

2.7082132683683358


2.71 > 1.708, therefore there is evidence to suggest that the ECMWF model predicts higher storm surges.

b. Using t-table for df = 25, we can see the that the bounds for p-value are 0.005 and 0.01 (0.005 <= p <= 0.01).

10.111.

a.

In [27]:
sam_n1_111 = 1055
sam_n2_111 = 1663
x1_111 = 530
x2_111 = 825
p1_111 = x1_111/sam_n1_111
p2_111 = x2_111/sam_n2_111
print(sam_n1_111*p1_111, sam_n1_111*(1-p1_111), sam_n2_111*p2_111, sam_n2_111*(1-p2_111))

530.0 525.0 825.0 838.0


n1p1 >= 5, n1(1-p1) >= 5, n2p2 >= 5, n2(1-p2) >=5, therefore the distribution of P1 - P2 is approximately normal.

b.

p1: proportion of carpoolers crossing the George Washington Bridge, p2: proportion of carpoolers using the Lincoln Tunnel

H0: p1 - p2 = 0, Ha: p1 - p2 > 0

alpha = 0.01, z_0.01 = 2.2363

In [28]:
pc_111 = (sam_n1_111/(sam_n1_111+sam_n2_111)*p1_111) + (sam_n2_111/(sam_n1_111+sam_n2_111)*p2_111)
z_111 = (p1_111 - p2_111)/math.sqrt(pc_111*(1-pc_111)*(1/sam_n1_111 + 1/sam_n2_111))
print(z_111)

0.3190210188791217


0.319 < 2.2363, therefore there is no evidence suggest that the proportion of carpoolers crossing the George Washington Bridge is greater than the proportion of carpoolers using the Lincoln Tunnel.

c. Using online p-value calculator using z = 0.319, we have p-value = 0.3749

10.139.

H0: var1 = var2, Ha: var1 != var2

alpha = 0.05 -> alpha/2 = 0.025

F_0.025,24,20 = 2.40756 & F_0.975,24,20 = 0.42969

In [29]:
n1_139 = 25
sam_var1_139 = 96.23
n2_139 = 21
sam_var2_139 = 194.02
F_139 = sam_var1_139/sam_var2_139
print(F_139)

0.49597979589733016


0.42969 < 0.49598 < 2.40756, therefore there is no evidence to suggest that the variability in checked baggage weight is different for these two airlines.